<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>855</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1306</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit_BaseDirectory"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Refresh">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTreeView" name="treeView_DirectoryTree">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="tableWidget_folder_property">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>250</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_backgrounds">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_foregrounds">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGraphicsView" name="graphicsView_foreground">
          <property name="minimumSize">
           <size>
            <width>960</width>
            <height>540</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>960</width>
            <height>540</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView_background">
          <property name="minimumSize">
           <size>
            <width>480</width>
            <height>270</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>480</width>
            <height>270</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView_diff">
          <property name="minimumSize">
           <size>
            <width>480</width>
            <height>270</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>480</width>
            <height>270</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
